manifests/k/KuroGames/PunishingGrayRaven/1.0/KuroGames.PunishingGrayRaven.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: KuroGames.PunishingGrayRaven
+PackageVersion: '1.0'
+InstallerType: nullsoft

```

Comment:
Although it says it installs successfully, in a clean VM, the install doesn't succeed. Perhaps there are different installer switches that are needed?